homework/js-exercises/Cat Walk/index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8" />
+    <title>Cat Walk</title>
+</head>
+
+<body>
+    <img style="position:absolute;" src="http://www.anniemation.com/clip_art/images/cat-walk.gif" />
+
+    <script>
+        let image = document.querySelector('img');
+        image.setAttribute('style', 'margin-left: 0px;');
+
+        const catWalk = () => {
+            positionOfCat = window.getComputedStyle(image).marginLeft.substring(0, window.getComputedStyle(image).marginLeft.length - 2);
```

Comment:
You should always use a variable keyword (var, const, let) when defining a variable.

Reply:
`window.getComputedStyle(image)` can be stored in a variable because it is used twice in this statement and also it is an "expensive" command